julia_src/http.jl

```diff
@@ -597,7 +598,7 @@ HTTP.register!(ROUTER, "POST", "/erp", erp)
 HTTP.register!(ROUTER, "POST", "/ghpghx", ghpghx)
 HTTP.register!(ROUTER, "GET", "/chp_defaults", chp_defaults)
 HTTP.register!(ROUTER, "GET", "/avert_emissions_profile", avert_emissions_profile)
-HTTP.register!(ROUTER, "GET", "/cambium_emissions_profile", cambium_emissions_profile)
```

Comment:
What's the reason for changing this url?

Reply:
the reason is that you could now also use this to get the grid clean energy fraction profile, not just the emissions profiles. Do you think it's better to leave it, even if the name wouldn't be 100% accurate?

Reply:
no no fine to change it then